.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "tabWidth": 2,
+  "semi": false,
+  "singleQuote": true
+}
```

Comment:
any reason you have an `.eslintrc.yml` _and_ a `.prettierrc`? You could add these in the `rules` section of the eslintrc. : )

Reply:
Prettier automatically enforces a strict style guide across the code base which ESLint does not. Here's a pretty good [write up](https://prettier.io/docs/en/why-prettier.html) on its benefits.